opentelemetry-sdk/src/opentelemetry/sdk/metrics/view.py

```diff
@@ -68,36 +68,64 @@ class View:
     def __init__(
         self,
         metric: InstrumentT,
-        aggregator: Aggregator,
+        aggregator: type,
+        aggregator_config: dict = None,
         label_keys: Sequence[str] = None,
```

Comment:
I think a lot of the typing needs to be `Optional`

Reply:
done, at least the ones I could find